### PR TITLE
Feature configure delay time out

### PR DESCRIPTION
**Pull request by SirWayne**
Added a new property to set the delay timeout for the DelayEventPushStrategy

Reviewers: @melanieretter @Jonkno

### CODE REVIEW

File: hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/ui/UIAutoConfiguration.java [hunk 1]
```diff
@@ -67,17 +68,24 @@ public UIEventProvider eventProvider() {
...
-     * 
+     * @param executorService
+     *            the general scheduler service
+     * @param eventBus
+     *            the ui event bus
+     * @param eventProvider
+     *            the event provider
+     * @param uiProperties
+     *            the ui properties
      * @return the provider bean
```
Comment (Jonkno) [inline hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/ui/UIAutoConfiguration.java]: A would also change the @return to: the event push strategy

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java [hunk 2]
```diff
@@ -270,12 +271,44 @@ public Cookie getCookie() {
         }
     }
 
+    /**
+     * Configuration of the UI event bus.
+     */
+    public static class Event implements Serializable {
+        private static final long serialVersionUID = 1L;
+
+        public static class Push implements Serializable {
```
Comment (Jonkno) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java]: Document this public class

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/push/DelayedEventBusPushStrategy.java [hunk 3]
```diff
@@ -72,17 +73,30 @@
...
+     *            the delay for the event forwarding. Every delay millisecond
+     *            the events are forwarded by this strategy
+     */
     public DelayedEventBusPushStrategy(final ScheduledExecutorService executorService, final UIEventBus eventBus,
-            final UIEventProvider eventProvider) {
+            final UIEventProvider eventProvider, final int delay) {
         this.executorService = executorService;
         this.eventBus = eventBus;
         this.eventProvider = eventProvider;
+        this.delay = delay;
```
Comment (kaizimmerm) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/push/DelayedEventBusPushStrategy.java]: I think this should be a long (needed for milliseconds).

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java [hunk 4]
```diff
@@ -270,12 +271,49 @@ public Cookie getCookie() {
...
+         * Configuration of the UI push.
+         *
+         */
+        public static class Push implements Serializable {
+            private static final long serialVersionUID = 1L;
+
+            /**
+             * The delay for the ui event forwarding.
+             */
+            private int delay = (int) TimeUnit.SECONDS.toMillis(2);
```
Comment (kaizimmerm) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java]: I think this should be a long (needed for milliseconds).

**Pull request merged**